client/my-sites/migrate/step-source-select.jsx

```diff
@@ -88,18 +92,22 @@ class StepSourceSelect extends Component {
 	};
 
 	render() {
-		const { targetSite, targetSiteSlug } = this.props;
+		const { targetSite, targetSiteSlug, translate } = this.props;
 		const backHref = `/import/${ targetSiteSlug }`;
-		const uploadHref = `/import/${ targetSiteSlug }?engine=wordpress`;
+		const uploadFileLink = `/import/${ targetSiteSlug }?engine=wordpress`;
 
 		return (
 			<>
-				<HeaderCake backHref={ backHref }>Import from WordPress</HeaderCake>
+				<HeaderCake backHref={ backHref }>{ translate( 'Import from WordPress' ) }</HeaderCake>
 				<CompactCard>
-					<CardHeading>What WordPress site do you want to import?</CardHeading>
+					<CardHeading>{ translate( 'What WordPress site do you want to import?' ) }</CardHeading>
 					<div className="migrate__explain">
-						Enter a URL and we'll help you move your site to WordPress.com. If you already have a
-						backup file, you can <a href={ uploadHref }>upload it to import content</a>.
+						{ translate(
+							"Enter a URL and we'll help you move your site to WordPress.com. If you already have a" +
+								"backup file, you can <a href='%(uploadFileLink)'>upload it to import content</a>.",
+							{ args: { uploadFileLink } }
```

Comment:
See comment about about using components.

Reply:
Thanks! Updated!

Reply:
Same again, I think Jon means we should extract the `a` into a component.